virttest/utils_v2v.py

```diff
@@ -691,7 +696,24 @@ def v2v_cmd(params):
 
     # Construct a final virt-v2v command
     cmd = '%s %s' % (V2V_EXEC, options)
-    cmd_result = process.run(cmd, timeout=v2v_timeout, verbose=True, ignore_status=True)
+    logging.debug(cmd)
+
+    # To deal with inputs during conversion
+    if inputs or (sasl_user and sasl_pwd):
+        v2v_log = os.path.join(data_dir.get_tmp_dir(), 'v2v_log')
+        v2v_session = aexpect.ShellSession(command=cmd,
+                                           output_func=utils_misc.log_line,
+                                           output_params=(v2v_log,))
+        remote.handle_v2v_prompts(v2v_session, sasl_user, sasl_pwd, timeout=600,
+                                  debug=True, input_line=inputs)
+        while v2v_session.is_alive():
+            time.sleep(10)
```

Comment:
What is the need of this wait (~indefinite, worstcase) loop?, 
As from the from previous call(remote.handle_v2v_prompts)
1. session would be dead already or
2. session would complete intended input/output exchanges.